egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -593,8 +595,11 @@ while [ $x -lt $num_iters ]; do
         k=$[$num_archives_processed + $n - 1]; # k is a zero-based index that we will derive
                                                # the other indexes from.
         archive=$[($k%$num_archives)+1]; # work out the 1-based archive index.
+        if [ $n -eq 1 ]; then
+          cache_io_opts=$cache_io_opts" --write-cache=$dir/cache.$[$x+1]" # opts for cache (storing pairs of nnet-computations and compute-requests) I/O
```

Comment:
sure. Actually I defined cache_io_opts earlier in the script so that problem shouldn't happen. But I should move the comment upward.

On Feb 10, 2016, at 8:13 PM, Daniel Povey <notifications@github.com> wrote:

> In egs/wsj/s5/steps/nnet3/lstm/train.sh:
> 
> > @@ -593,8 +595,11 @@ while [ $x -lt $num_iters ]; do
> >          k=$[$num_archives_processed + $n - 1]; # k is a zero-based index that we will derive
> >                                                 # the other indexes from.
> >          archive=$[($k%$num_archives)+1]; # work out the 1-based archive index.
> > +        if [ $n -eq 1 ]; then
> > +          cache_io_opts=$cache_io_opts" --write-cache=$dir/cache.$[$x+1]" # opts for cache (storing pairs of nnet-computations and compute-requests) I/O
> Also please put $cache_io_opts inside the quotes.
> 
> —
> Reply to this email directly or view it on GitHub.
>